deps/ledger-zxlib/dockerized_build.mk

```diff
@@ -42,6 +42,15 @@ ifeq ($(TARGET_DEVICE), NANO_X)
     TEST_DEVICE=nanox
     SPECULOS_SDK=2.0.2
 endif
+ifeq ($(TARGET_DEVICE), NANO_SP)
```

Comment:
I'm concerned that it's confusing that in some places we are using "P" and in others we are using "2" to designate the Nano S Plus.